server/db/models/job.js

```diff
@@ -25,7 +25,14 @@ const Job = db.define(
       },
     },
     status: {
-      type: ENUM('paid', 'unpaid', 'cancelled', 'completed'),
+      type: ENUM(
+        'funded',
+        'pending',
+        'pendingVerfication',
```

Comment:
copied these model changes in my PR to avoid conflicts, just need to fix the typo on verification before merging